send-transaction-service/src/transaction_client.rs

```diff
@@ -285,26 +265,28 @@ impl TpuClientNextClient {
                 my_tpu_address,
                 tpu_peers,
             };
-        let config = {
-            let bind_socket = bind_socket
-                .try_clone()
-                .expect("Cloning bind socket should always finish successfully.");
-            Self::create_config(bind_socket, identity, leader_forward_count as usize)
-        };
+        let config = Self::create_config(bind_socket, identity, leader_forward_count as usize);
+
         let scheduler = ConnectionWorkersScheduler::new(Box::new(leader_updater), receiver);
         // leaking handle to this task, as it will run until the cancel signal is received
         runtime_handle.spawn(scheduler.get_stats().report_to_influxdb(
             "send-transaction-service-TPU-client",
             METRICS_REPORTING_INTERVAL,
             cancel.clone(),
         ));
-        let handle = runtime_handle.spawn(scheduler.run(config, cancel.clone()));
+        // TODO(klykov): store handle in structure later, when move the logic for update to separate structure.
```

Comment:
It is possible to move the logic for implementing NotifyUpdate to a separate structure that will have only corresponding `Sender`. I tried this in a separate branch, doesn't really simplify the code or makes any better except of creating another abstraction. So probably will not go this way.